scania-digital-design-system/tegel
# build: config

**Describe pull-request**  
Updated tsconfig for root, core and react to differentiate the different jsx options.

**Solving issue**  
Fixes: -

**How to test**  
1. Check out branch
2. Run `npm run build-all`
3. Check the build for errrors
